config.toml

```diff
@@ -12,3 +12,25 @@ copyright = '© Copyright 2023 The Academic Folio Contributors'
   description = "Albert Einstein's personal website powered by the Academic Folio theme"
 
   footerLastUpdated = true
+
```

Comment:
I moved the config files to a directory structure: https://gohugo.io/getting-started/configuration/#configuration-directory

Reply:
This is Fixed.